src/runtime_src/core/pcie/emulation/cpu_em/generic_pcie_hal2/system_swemu.cxx

```diff
@@ -58,6 +58,13 @@ get_total_devices(bool is_user) const
   return {nd,nd};
 }
 
+std::tuple<uint16_t, uint16_t, uint16_t, uint16_t>
+system::
+get_bdf_info(device::id_type id, bool is_user) const
+{
+  return std::make_tuple(0, 0, id, (is_user ? 1 : 0));
+}
```

Comment:
Really?  Do we care to have this special implementation?  There is no bdf in swemu.